astropy/io/fits/tests/test_hdulist.py

```diff
@@ -178,6 +178,39 @@ def test_append_groupshdu_to_non_empty_list(self):
         with pytest.raises(ValueError):
             hdul.append(hdu)
 
+    @pytest.mark.parametrize(
+        ["image", "do_not_scale"],
+        [
+            ["scale.fits", True],
+            ["o4sp040b0_raw.fits", True],
+            ["fixed-1890.fits", True],
+            ["scale.fits", False],
+            ["o4sp040b0_raw.fits", False],
+            ["fixed-1890.fits", False],
+        ],
+    )
```

Comment:
I think you could do something like that to get all combinations:
```
@pytest.mark.parametrize("image", ["scale.fits", ...])
@pytest.mark.parametrize("do_not_scale", [True, False])
```

Reply:
Thank you @saimn ... I didn't know that!